### DISCUSSION TITLE
nginx user does not exist

### DISCUSSION THREAD

**Discussion by bestrouting** [nautobot/nautobot#6093] [Q&A]
Hello, 

I am configuring an http server using nginx based on the Nautobot Documentation.
An error message below is shown when "sudo usermod -aG nautobot nginx" is executed.
    usermod: user 'nginx' does not exist
All services nautobot, nautobot-worker, nautobot-scheduler and nginx are working with no issue.
I got a "Bad request 400" when I access the nautonbot.
I am thinking of the bad request error might have something to do with the error "nginx does not exit".
Could someone please advise me what is wrong about "nginx' user?

**Comment by digital-fortress** [comment]
Nginx may be under a user www-data. Check the Nginx configuration to find the user:

cat /etc/nginx/nginx.conf | grep user


If Nginx is running under the www-data user, you should add www-data to the nautobot group:

sudo usermod -aG nautobot www-data

verify-
ps aux | grep nginx

groups www-data

**Reply by bestrouting** [reply]
Hi,
www-data has just been added to the nautobot with no issue.
But I still get "Bad Request (400)".
I built it from scratch multiple times but no luck.

**Reply by digital-fortress** [reply]
This solution worked for me , so I am helpless.

**Reply by glennmatthews** [reply]
Is there any context on the 400 response or is that literallly the only information available?

**Reply by bestrouting** [reply]
I have found out that the environmental values was configured wrong.
The nautobot started working correctly after it was fixed.
"sudo usermod -aG nautobot www-data" should be updated because without this command, nautobot is not working at all.
Thank you.

**Reply by ernestoherrerab** [reply]
This also worked for me

**Comment by glennmatthews** [comment]
Sounds like a documentation update may be needed. @bestrouting Can you confirm what OS you're running on?

**Reply by bestrouting** [reply]
I am running it on Ubuntu 22.04.4 LTS.
